File: api/src/test/java/work/beltran/discogsbrowser/api/SearchTest.java
//package work.beltran.discogsbrowser.api;
//
//import org.junit.Test;
//
//import rx.observers.TestSubscriber;
//import work.beltran.discogsbrowser.api.model.SearchResults;
//
///**
// * Created by Miquel Beltran on 8/27/16
// * More on http://beltran.work
// */
//public class SearchTest extends DiscogsServiceTest {
//
//    private static final String QUERY = "String";
//    private static final String TYPE = null;
//    private static final String FORMAT = null;
//    private static final String BARCODE = null;
//
//    @Test
//    public void testSearch() throws Exception {
//        TestSubscriber<SearchResults> subscriber = TestSubscriber.create();
//        service.search(QUERY, TYPE, FORMAT, BARCODE).subscribe(subscriber);
//        subscriber.assertNoErrors();
//        subscriber.assertCompleted();
//    }
//}

File: data-api/src/test/java/work/beltran/discogsbrowser/api/SearchTest.kt
package work.beltran.discogsbrowser.api

import org.junit.Test

/**
 * Created by Miquel Beltran on 8/27/16
 * More on http://beltran.work
 */
class SearchTest : DiscogsServiceTest() {

    @Test
    fun `search using query`() {
        val subscriber = service.search(QUERY, null, null, null).test()
        subscriber.assertNoErrors()
        subscriber.assertComplete()
    }

    companion object {
        private const val QUERY = "String"
    }
}
